sklearn/model_selection/tests/test_split.py

```diff
@@ -2054,3 +2054,19 @@ def test_no_group_splitters_warns_with_groups(cv):
 
     with pytest.warns(UserWarning, match=msg):
         cv.split(X, y, groups=groups)
+
+
+def test_Stratified_mandatory_y_argument():
```

Comment:
We generally don't use capital letters in test function names, I would use something like this:
```suggestion
def test_stratified_splitter_without_y():
```